src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_constants.py

```diff
@@ -151,7 +151,15 @@ def version_str(v):
 except AttributeError:
     PY_IMPL_NAME = ''
 
-SUPPORT_GEVENT = os.getenv('GEVENT_SUPPORT', 'False') == 'True'
+SUPPORT_GEVENT = os.getenv('GEVENT_SUPPORT', 'False') in ('True', 'true', '1')
+
+GEVENT_SUPPORT_NOT_SET_MSG = os.getenv(
```

Comment:
Note: I made the message configurable through an environment variable, but I `ptvsd` should probably set it on `force_pydevd.py` which also sets other constants `pydevd_constants` if needed.